src/e2e/helpers/visit.js

```diff
@@ -0,0 +1,26 @@
+import nightmare from 'nightmare';
+import url from 'url';
+
+const BASE_URL = url.format({
```

Comment:
You can use DefinePlugin to set up these ENV vars when using a seperate config file for E2E!